14/README.md

```diff
@@ -134,28 +136,34 @@ For this BEP, we analyze a round-robin connection strategy, similar to [Round-ro
 In this scenario, every request is forwarded to a different node, distributing the load to all the nodes, and allowing recovering from connection errors by forwarding the failed request to another healthy node.
 
 This strategy needs to keep track of the following values:
-- `E`, the list of BigchainDB endpoints passed at initialization time (we use the Python notation `len(L)` to get the length of the list).
-- `i`, the index of the BigchainDB node to use for a request (`0 ≤ i < len(L)`).
-- `T`, a list of timestamps. It has the same length of `L`. On initialization, all elements contain `0`. It is used to track the availability of the nodes. We will see in a second how it is used.
-- `DELAY`, in case a node is unreachable, how much milliseconds we should wait. This is used in combination with `T`.
+
+- `E`, the list of BigchainDB endpoints passed at initialization time.
+- `timeout`, user-specified timeout, by default there is no timeout.
```

Comment:
If a developer initializes the driver with a wrong endpoint, the driver will forever fail the request. Not having a default timeout might make the code hang forever.

I'd put a default timeout (just like any socket connection) of [20 seconds](http://www.sekuda.com/overriding_the_default_linux_kernel_20_second_tcp_socket_connect_timeout), that can be initialized to `None` if the user wants to **not** have a timeout.